docs/source/howto/data.rst

```diff
@@ -260,6 +260,338 @@ Finding data
 Organizing data
 ===============
 
+Groups
+------
+Group is a tool to organize the nodes of the provenance graph into sub sets.
+Any number of groups can be created and each group can contain any number of nodes of any type.
+
+Typically, you want to put multiple nodes into a group because they share some common property, and through the group they can easily be referenced.
+Unlike nodes, groups can be modified at any time.
+Here we profide a list of typical operations that may be performed with groups:
+
+.. note::
+    Any deletion operation related to groups won't affect the nodes themselves.
+    For example if you delete a group, the nodes that belonged to the group will remain in the database.
+    The same happens if you delete nodes from the group -- they will remain in the database but won't belong to the group anymore.
```

Comment:
This should be moved to the section on deleting groups

Reply:
moved.